tests/test_jwt.py

```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/python2.4
-#
-# Copyright 2014 Google Inc. All rights reserved.
+# Copyright 2015 Google Inc. All rights reserved.
```

Comment:
Ahh I see. I only did it because I was there. Will revert (it'll make the diff look nicer)